core/src/main/java/org/ironjacamar/core/connectionmanager/pool/dflt/DefaultManagedConnectionPool.java

```diff
@@ -165,4 +170,99 @@ else if (cl.getState() == DESTROY)
       }
       listeners.clear();
    }
+
+   /**
+    * Prefill
+    */
+   @Override public void prefill()
+   {
+      if (this.credential.equals(pool.getPrefillCredential()))
+      {
+         PoolFiller.fillPool(new FillRequest(this, pool.getConfiguration().getInitialSize()));
```

Comment:
This should be getMinSize() -- the InitialPoolSize() should be used in the constructor